WebExample/webpack.config.js

```diff
@@ -10,5 +10,6 @@ module.exports = async function (env, argv) {
     },
     argv,
   );
+  config.resolve.fallback = {crypto: false};
```

Comment:
Why do we need this change?

Reply:
This change is fixing this error in build-web: 
<img width="1516" alt="Screenshot 2024-11-21 at 12 23 14" src="https://github.com/user-attachments/assets/c713c30c-d81e-4db3-b063-0c9c3fd7055f">